chainer/functions/connection/convolution_nd.py

```diff
@@ -328,18 +328,53 @@ def convolution_nd(x, W, b=None, stride=1, pad=0, use_cudnn=True,
       spatial dimensions, respectively.
     - :math:`k_1, k_2, ..., k_N` are the size of each axis of the filters,
       respectively.
+    - :math:`l_1, l_2, ..., l_N` are the size of each axis of the output's
+      spatial dimensions, respectively.
+    - :math:`p_1, p_2, ..., p_N` are the size of each axis of the spatial
+      padding size, respectively.
+
+    Then the ``convolution_nd`` function computes correlations between filters
+    and patches of size :math:`(k_1, k_2, ..., k_N)` in ``x``.
+    Note that correlation here is equivalent to the inner product between
+    expanded tensors.
+    Patches are extracted at positions shifted by multiples of ``stride`` from
+    the first position ``(-p_1, -p_2, ..., -p_N)`` for each spatial axis.
+
+    Let :math:`(s_1, s_2, ..., s_N)` be the stride of filter application.
+    Then, the output size :math:`(l_1, l_2, ..., l_N)` is determined by the
+    following equations:
+
+    .. math::
+
+       l_n = (d_n + 2p_n - p_n) / s_n + 1 \ \ (n = 1, ..., N)
```

Comment:
It seems strange to me because it does not depend on `k_n`. Maybe second `p_n` should be replaced with `k_n`.